### PR TITLE
chore: roll to ToT

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: tests/async/test_page.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ async def test_close_should_reject_all_promises(context):
     assert "Protocol error" in exc_info.value.message
 
 
-async def test_closed_should_not_visible_in_context_pages(context):
+async def test_closed_should_not_visible_in_context_pages(context, browser):
```
Comment (mxschmitt) [inline tests/async/test_page.py:33]: whats the reason for that?

File: tests/conftest.py [hunk 2]
```diff
@@ -48,8 +48,10 @@ def assetdir():
 
 @pytest.fixture(scope="session")
 def launch_arguments(pytestconfig):
+    print(pytestconfig.getoption("--browser-channel"))
```
Comment (mxschmitt) [inline tests/conftest.py:51]: do you think this has potential to get adopted in pytest-playwright? I think so.

**Review by mxschmitt — APPROVED** [review]

File: tests/async/test_page.py (hunk 1 above)
Comment (pavelfeldman) [reply tests/async/test_page.py:33]: i was testing the version to see if channel works... reverting

File: tests/conftest.py (hunk 2 above)
Comment (pavelfeldman) [reply tests/conftest.py:51]: Yes, I think so.

**Pull request merged**